tests/drivers/stepper/stepper_drv8424_api/src/main.c

```diff
@@ -0,0 +1,501 @@
+/*
+ * Copyright 2024 Navimatix GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/drivers/gpio.h"
```

Comment:
#include <zephyr/drivers/gpio.h>, same for the other zephyr includes as well :-)